verifier/truststore/errors.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package truststore
+
+// ErrorTrustStore is used when accessing specified trust store failed
+type ErrorTrustStore struct {
+	WrappedError error
+}
```

Comment:
Oh, I see. In `errors.go` under `notation` package, we already have all the types defined as `ErrorXXX` though. Should we keep the style so that they are consistent?